### PR TITLE
[device/celestica]: Add support for fan/voltage event to questone2bd get_change_event api

### PR DESCRIPTION
**Pull request by mudsut4ke**
**- What I did**
- Implement polling method to get fan insert/remove event in questone2bd platform api
- Implement polling method to get voltage over threshold event in questone2bd platform api

**- How I did it**
- Modify get_change_event() in platform chassis. 
- Add event.py to store VoltageEvent, FanEvent and SfpEvent class

**- How to verify it**
- Install voltage driver (https://github.com/SONIC-DEV/sonic-buildimage/pull/197)
- Run test script and set an event: [test_event.zip](https://github.com/SONIC-DEV/sonic-buildimage/files/4918227/test_event.zip)
- Test log: [test_event.log](https://github.com/SONIC-DEV/sonic-buildimage/files/4918230/test_event.log)

### CODE REVIEW

File: device/celestica/x86_64-cel_questone2bd-r0/sonic_platform/chassis.py [hunk 1]
```diff
@@ -304,78 +259,82 @@ def get_status(self):
...
+            chk_sfp = sfp_event.check_all_port_interrupt_event()
+            int_sfp = sfp_event.update_port_event_object(
+                chk_sfp, int_sfp) if chk_sfp else int_sfp
+            int_fan = fan_event.check_fan_status(cur_fan_state, int_fan)
+            int_volt = voltage_event.get_voltage_int()
+
+            current_milli_time = int(round(time.time() * 1000))
+            if timeout == 0 and (int_sfp or int_fan or int_volt) or \
+                    timeout != 0 and (current_milli_time - start_milli_time > timeout):
+                break
```
[the remark this answers is not in the document]
Comment (mudsut4ke) [reply device/celestica/x86_64-cel_questone2bd-r0/sonic_platform/chassis.py:331]: In that case, How can capture multiple event such as {'fan':{'0':'0', '2':'1'}, 'sfp':{'11':'0'}, 'voltage':{'U20':'1', 'U1':'0'}} ?

File: device/celestica/x86_64-cel_questone2bd-r0/sonic_platform/chassis.py [hunk 2]
```diff
@@ -304,78 +259,82 @@ def get_status(self):
...
+
+            time.sleep(POLL_INTERVAL)
```
[the remark this answers is not in the document]
Comment (mudsut4ke) [reply device/celestica/x86_64-cel_questone2bd-r0/sonic_platform/chassis.py:333]: I'm agree with you

[the remark this answers is not in the document]
Comment (mudsut4ke) [reply device/celestica/x86_64-cel_questone2bd-r0/sonic_platform/chassis.py:333]: I'm agree with this

File: device/celestica/x86_64-cel_questone2bd-r0/sonic_platform/chassis.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (pphuchar) [reply device/celestica/x86_64-cel_questone2bd-r0/sonic_platform/chassis.py:331]: We are only interested in the event that occurs until the function called to lines 322-326.
If those events really happened at the same time, they will be recorded and return in the same dictionary.

**Comment by mudsut4ke** [comment]
@pphuchar , Please review https://github.com/SONIC-DEV/sonic-buildimage/pull/196/commits/a0b6ecfb344949c68541869ff4f76138618c46be

**Pull request merged**